frontend/src/app/features/details/SiteDetails.css

```diff
@@ -183,6 +183,12 @@
   border-bottom: 1px solid var(--surface-border-light, #d8d8d8);
 }
 
+@media (max-width: 642px) {
```

Comment:
Can we use bootstrap break points instead of custom  width setting ?

Reply:
done